color/src/colorspace.rs

```diff
@@ -342,6 +372,13 @@ fn lch_to_lab([l, c, h]: [f32; 3]) -> [f32; 3] {
     [l, a, b]
 }
 
+/// The cylindrical version of the [Oklab] color space.
```

Comment:
From my perspective, it depends whether the use is referring to the type or the concept of the type. For example, I have used "[Drains](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.drain) this ..." in `xilem_core`, when I was referring to the concept of draining, rather than the specific implementation.

In this case, we are referring to the colour space implemented by the linked type, so IMO no backticks is fine (but this isn't a hill I'll die on)

Reply:
For complete consistency, I think this should then be ``[Oklab](`Oklab`)``.